### ISSUE TITLE
Erase-Geometry Option?

### ISSUE THREAD

**Issue by jhenzel** [arnaudleclerc/AzureMapsControl.Components#73]
Browsing through the source code I notice that no Erase-Geometry option is available as is on the pure JavaScript side.  This sample from Microsoft illustrates it

[https://github.com/Azure-Samples/AzureMapsCodeSamples/blob/vnext/Samples/Drawing%20Tools%20Module/Drawing%20toolbar%20options/Drawing%20toolbar%20options.html]

In lieu of this option no available on the map, what do you recommend for deleting polygons, etc. drawn on the map?

**Comment by arnaudleclerc** [comment]
I assume you added your shapes or features to a data source, did you? The datasource itself has methods to remove objects from a layer.

Is this what you're looking for?

**Comment by jhenzel** [comment]
Not quite. Requirement is for use to select an area of interest using the tools.

**Comment by PhilpD** [comment]
I have a similar requirement. It would be good to have the DrawingManager available. so you could pass in your data layer and get the built in editing etc.

**Comment by sameerconsforc** [comment]
I'm dealing with a similar issue—trying to remove a geometry point created using DrawingToolbarOptions in Azure Maps. I'm struggling to figure out how to activate or utilize the delete button feature in the code. This problem is really holding me back in advancing my project. How can I clear all the geometries?
